pm_yaml_config/test-data/clusters.txt

```diff
@@ -0,0 +1,73 @@
+
```

Comment:
Most of the files in this `test-data` folder are not relevant to this tool and should be removed.

Reply:
removed